catroid/src/main/res/values/strings.xml

```diff
@@ -1788,6 +1788,8 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_function_letter_parameter">(1,\'hello world\')</string>
     <string name="formula_editor_function_join">join</string>
     <string name="formula_editor_function_join_parameter">(\'hello\',\' world\')</string>
+    <string name="formula_editor_function_join2">join2</string>
```

Comment:
```suggestion
    <string name="formula_editor_function_join3">join</string>
```
No need to give it a different visual name from a users perspective. That's also how it was specified in the ticket.